build.gradle

```diff
@@ -4,7 +4,7 @@ plugins {
 	id 'eclipse'
 	id 'maven-publish'
 	id("org.cadixdev.licenser") version "0.5.0"
-    id("org.quiltmc.loom") version "0.7.12"
+    id("org.quiltmc.loom") version "0.7.13"
```

Comment:
Could you make sure you set the buildscript to indent with tabs in eclipse?

Reply:
What the heck? I have eclipse set to use tabs everywhere as it's my preference as well. Looking at the diff who knows how long that's been spaces. Fixed it anyway.